etna/models/holt_winters.py

```diff
@@ -372,9 +398,12 @@ def predict_components(self, df: pd.DataFrame) -> pd.DataFrame:
         model = self._model
         fit_result = self._result
 
-        if fit_result is None or model is None:
+        if fit_result is None or model is None or self._train_freq is None:
             raise ValueError("This model is not fitted!")
 
+        if df["timestamp"].min() < self._first_train_timestamp or df["timestamp"].max() > self._last_train_timestamp:
```

Comment:
We actually only have test for the second case, but we don't check this one `df["timestamp"].min() < self._first_train_timestamp`

Reply:
Added